cpp/ql/lib/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.3.0
+
+### Deprecated APIs
+
+* The `BarrierGuard` class has been deprecated. Such barriers and sanitizers can now instead be created using the new `BarrierGuard` parameterized module.
+
+### New Features
+
+* An `isBraced` predicate was added to the `Initializer` class which holds when a C++ braced initializer was used in the initialization.
```

Comment:
```suggestion
```
Deleting this manually.

The duplication happened because the change note file was merged both on main and the old release branch, so the file on main initially survived deletion.